modules/log-upgrade/6x-logging-upgrading-clo.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/cluster-logging-upgrading.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-upgrading-clo_{context}"]
+= Updating the {clo}
+
+To update the {clo} to a new major release version, you must modify the update channel for the Operator subscription.
+
+.Prerequisites
+
+* You have installed the {clo}.
+* You have administrator permissions.
+* You have access to the {product-title} web console and are viewing the *Administrator* perspective.
+
+.Procedure
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Select the *openshift-logging* project.
+
+. Click the *Red Hat OpenShift Logging* Operator.
+
+. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link.
+
+. In the *Change Subscription Update Channel* window, select the latest major version update channel, *stable-6.x*, and click *Save*. Note the `cluster-logging.v6.y.z` version.
+
+. Wait for a few seconds, and then go to *Operators* -> *Installed Operators* to verify that the {clo} version matches the latest `cluster-logging.v5.9.<z>` version.
```

Comment:
If it's being upgraded to Logging 6, then, the version when it's being verified should be v6.y.z and not 5.9.z

Reply:
fixed